classes/NPRAPIWordpress.php

```diff
@@ -501,30 +522,48 @@ function send_delete( $api_id ) {
     }
 
   /**
-   * 
+   *
    * This function will check a story to see if there are transcripts that should go with it, if there are
    * we'll return the transcript as one big strang with Transcript at the top and each paragraph separated by <p>
    * 
-   * @param  $story
+   * @param string $story
+   * @return string
    */
     function get_transcript_body( $story ) {
         $transcript_body = "";
-        if ( ! empty( $story->transcript ) ) {
+        if ( ! empty( $story->transcript ) && is_array( $story->transcript ) ) {
             foreach ( $story->transcript as $transcript ) {
-                if ( $transcript->type == 'api' ) {
-                    $response = wp_remote_get( $transcript->value );
-                    if ( !is_wp_error( $response ) ) {
-                        $transcript_body .= "<p><strong>Transcript :</strong><p>";
```

Comment:
@benlk As you can see the old code they did not close `<p>`s either!  

I was trying to limit the scope of my changes as much as possible to just solve the solve the _"Trying to get property of non-object"_ errors because I know it is easier to accept changes when they are small.  

That said, I'll add another commit that should add `</p>` closing tag.